examples/sglang/utils/wideep/gap_bench.sh

```
@@ -0,0 +1,30 @@
+MODEL=deepseek-ai/DeepSeek-R1
+OSL=5
+INPUT_FILE=<path-to-input-file>
```

Shellcheck (0.10.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)